### PR TITLE
Bug fixes

### LINKED ISSUE #46: Conditional routing
As a dev, I want the user to be directed to the style Survey the first time they click "+ Subscription".  If it is not the first time, then the button should direct the user to their Subscriptions page.

### PR DESCRIPTION
**Pull request by bruce-gordon**
#### What does  this PR do?
- Fixes bugs related to style and conditional rendering
- Cleans up code, removes console logs and unneeded comments

#### Where should the reviewer start?
- web/src/modules/survey/Question.js
- web/src/modules/crate/Item.js
- web/src/modules/user/Survey.js

#### How should this be manually tested?
- Pull down branch, select a crate, complete survey, view profile
- Once survey has been completed, navigate back to Crates and select another crate.  You will navigate to Subscriptions instead of Survey.

#### Any background context you want to provide?
- These updates are aligned with our discussions and wireframe.

#### What are the relevant tickets?
- closes #46

### CODE REVIEW

File: code/web/src/modules/user/Profile.js [hunk 1]
```diff
@@ -14,19 +14,10 @@ import { grey, grey2 } from '../../ui/common/colors'
 // App Imports
 import userRoutes from '../../setup/routes/user'
 import { logout, getStyle } from './api/actions'
-// import SurveyItem from '../survey/Item'
```
Comment (GeorgieGirl24) [inline code/web/src/modules/user/Profile.js:17]: great job removing the unneeded comments

**Review by GeorgieGirl24 — APPROVED** [review]

File: code/web/src/modules/user/api/actions.js [hunk 2]
```diff
@@ -175,37 +173,3 @@ export function getGenders() {
     }))
   }
 }
-
-// export function getStyle(isLoading = true) {
-//   return dispatch => {
```
Comment (Callbritton) [inline code/web/src/modules/user/api/actions.js:180]: Thank you for cleaning up this code we didn't end up utilizing!

**Review by Callbritton — APPROVED** [review]
Excellent job, team!

File: code/web/src/modules/user/api/actions.js [hunk 3]
```diff
@@ -72,7 +72,6 @@ export function login(userCredentials, isLoading = true) {
 }
 
 export function getStyle (userDetails, isLoading = true) {
-  console.log("peanut butter falcon")
```
Comment (atownse) [inline code/web/src/modules/user/api/actions.js:75]: Good catch on the console logs

File: code/web/src/modules/survey/Question.js [hunk 4]
```diff
@@ -14,25 +14,37 @@ class Question extends PureComponent {
...
+
   handleClick = async (e) => {
     let choice = e.target.id
     await this.setState({ choice: parseInt(choice)})
     this.props.saveSelection(`q${this.props.id}`, this.state.choice)
   }
 
-  choices = this.props.choices.map(choice => {
+  showSelected = (choiceNum) => {
+    const start = { display: 'flex', flexDirection: 'column', alignItems: 'center', borderRadius: '25px', padding: '0.5em', width: '50%'}
```
Comment (atownse) [inline code/web/src/modules/survey/Question.js:25]: Maybe find a way to clean up styling in the future

**Review by atownse — APPROVED** [review]

**Pull request merged**